packages/search-ui/src/SearchDriver.js

```diff
@@ -235,13 +250,29 @@ export default class SearchDriver {
         if (this.requestSequencer.isOldRequest(requestId)) return;
         this.requestSequencer.completed(requestId);
 
+        // Results paging start & end
+        const { totalResults } = resultState;
+        const start =
```

Comment:
haha we were renaming `resultTerm` to `resultSearchTerm` anyway so I thought I'd keep the renaming scheme consistent :trollface: 

No strong opinion honestly, generally I also prefer to be a bit more specific, but `current` was kind of throwing me off (`current`/`start`/`end` feel like they all kinda go together). So I thought I'd split the difference and keep both start and pagingStart in the PR so that the move was a little clearer to follow.

Reply:
Yeah, no problem, your explanation makes total sense! I like that you decided to be consistent 😄